src/CTA.WebForms/Services/TaskManagerService.cs

```diff
@@ -19,14 +19,16 @@ public class TaskManagerService
         public enum TaskState { Active, Waiting }
         public enum TaskStallTimeout { None = 0, Short = 100, Medium = 1000, Long = 10000 }
```

Comment:
Can we rename this to include the unit of time? E.g. `TaskStallTimeoutInSeconds`/`Ms`

A comment with unit of time would also be helpful.

Reply:
This is a good idea, will add